src/App.css

```diff
@@ -112,18 +115,15 @@
   font-weight: bold;
 }
 
-.book-card p {
-  margin: 8px 0;
+.book-card .leftButton {
   color: #4386bf;
-}
-
-.book-card button {
-  background-color: #4386bf;
-  color: white;
   border: none;
-  padding: 8px 12px;
-  border-radius: 4px;
-  cursor: pointer;
+  width: 4.563rem;
+  height: 1.188rem;
+  font-weight: 300;
+  letter-spacing: normal;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
```

Comment:
Good job styling your app, Unfortunately, there are some small changes to make your design match the given Zeplin design:
- [ ] Please remove the welcome message on the top of the page because it does not exist in the original design.

- [ ] For the book cards, please add the correct width value and also remove the border lines (the black border around the whole cards and the red button border between each book card) because it does not exist in the original design.

- [ ] For the book title, Please add the correct font family `RobotoSlab-Bold`.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/hydaline-code/Bookstore/assets/68796232/6a1c9e94-16bd-4d86-8706-f505e8b81c12)  | ![image](https://github.com/hydaline-code/Bookstore/assets/68796232/bc8f61fa-e607-40b2-adb2-ba273219ec5a)  |